client/my-sites/customer-home/cards/tasks/go-mobile/index.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import userAgent from 'lib/user-agent';
+import Task from '../task';
+import AppsBadge from 'blocks/get-apps/apps-badge';
+
+/**
+ * Image dependencies
+ */
+import appleStoreLogo from 'assets/images/customer-home/apple-store.png';
+import googlePlayLogo from 'assets/images/customer-home/google-play.png';
+
+const GoMobile = () => {
+	const translate = useTranslate();
+	const { isAndroid } = userAgent;
+
+	const actionButton = isAndroid ? (
```

Comment:
There are some devices that are neither Android nor iOS (i.e. old Nokia phones using Windows Mobile). @danhauk should we display both badges on those? 

Actually, it might be better to actually don't display this task at all, so in that case we'll need to handle it server-side in D43129-code (cc @kwight).

Reply:
Based on the code shared by @kwight , if those devices are neither, they are considered as 'Desktop', so the Task won't be displayed anyway, right?

Reply:
> it might be better to actually don't display this task at all, so in that case we'll need to handle it server-side
That was my idea when I wrote the component...

Reply:
How about two string uids that map to the same component but with a different device prop that we'd map in a location component? +1 for not needing a client check.

So the server would pass back either:
`home-task-go-mobile-android` 
`home-task-go-mobile-ios`

And in this client PR we can add a prop(s) that are passed through. (We can drop the userAgent check). Usage might look like the following:

```jsx
<GoMobile isAndroid />
<GoMobile isIoS />
```  

Or if we want one prop:
```jsx
<GoMobile device="android" />
<GoMobile device="ios" />

```

Reply:
Note that my suggestion is non-blocking, as we can always tidy this up in the next PR when we wire up the component display to the API.

Reply:
I've updated D43129 to return `home-task-go-mobile-android` or `home-task-go-mobile-ios` in a `VIEW_GO_MOBILE` view as appropriate, and nothing (via some other valid view) if not.

Reply:
@kwight @gwwar It makes sense. I implemented it this way.